.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
This file does not seem to be related to the gem and we probably should keep it out the repository, just like we do with `.idea` files.

Reply:
I think it's not the same.
editorconfig is setup for all editors (some require plugin for it)
Idea is only for intelliJ (?)

as a reference see:
https://github.com/ruby/ruby/blob/master/.editorconfig

It's more like rubocop setttings for development purposes.

Reply:
Ok, that seems like pretty cool idea, haven't seen that before.